### DISCUSSION TITLE
Deprecate the media-text block

### DISCUSSION THREAD

**Discussion by aristath** [WordPress/gutenberg#40924] [Ideas]
For the past few weeks, I've been thinking about the media-text block, how it is used and why we have it.
I can't think of anything this block does that can't be achieved with a combination of other blocks...
Could we perhaps work on a deprecation plan for that block, migrating it to become a variation of other blocks? 🤔 
If a feature is missing from columns for example in order to achieve the same behavior, then adding said behavior to the columns would be better long-term than keeping around the media-text block.

**Comment by carolinan** [comment]
Removing the block from the inserter would mean that existing blocks can continue working as before, and new block patterns without the accessibility issues would be available instead.

**Reply by gziolo** [reply]
We have some deprecated blocks present in the code, but they are hidden from the inserter. We could always follow that.

**Comment by bgardner** [comment]
The only reason I use the Media & Text block is to ensure consistency on mobile devices when it comes to displaying an image and text. For instance, on the [Frost home page](https://frostwp.com/), there's a section that alternates Text/Media, Media/Text, Text/Media. With the Media & Text block, all instances show Media/Text. If I were to use a column block, one instance would show Text/Media, which would not be ideal.

**Reply by patric-boehner** [reply]
I have this issue on a current design as well though for now have decided to live with the order issue.

**Comment by DaisyOlsen** [comment]
I agree that if a block, intended for a very specific use-case, can be created with component blocks in patterns it makes sense to deprecate. Having the patterns available to replace the deprecated block would be a good idea to help with transition for users.

**Comment by audunmb** [comment]
An old issue, but just to weigh in: Media and Text uses grid, while Columns uses flexbox. So until there's a grid block available in core, there's no replacement.
